src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -389,6 +389,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 #End Region
 
 #Region "Buffer helpers"
+
```

Comment:
The build I have is `Debug` so it isn't optimised. It the IL it is a method call.
```
	IL_00ce: nop
	IL_00cf: ldarg.0
	IL_00d0: ldloc.1
	IL_00d1: ldc.i4.2
	IL_00d2: add
	IL_00d3: ldstr "CDATA["
	IL_00d8: call instance bool Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.Scanner::NextAre(int32, string)
	IL_00dd: ldc.i4.0
	IL_00de: ceq
	IL_00e0: stloc.2
	IL_00e1: ldloc.2
	IL_00e2: brtrue.s IL_00f2
```

Since its been extracted out we can "play" with different internal implementations.